plugins/outputs/quix/quix.go

```diff
@@ -0,0 +1,130 @@
+// quix.go
+package quix
+
+import (
+	"crypto/sha256"
+	"strings"
+
+	"github.com/IBM/sarama"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+// Quix is the main struct for the Quix plugin
+type Quix struct {
+	Brokers        []string `toml:"brokers"`
+	Topic          string   `toml:"topic"`
+	Workspace      string   `toml:"workspace"`
+	AuthToken      string   `toml:"auth_token"`
+	APIURL         string   `toml:"api_url"`
+	TimestampUnits string   `toml:"timestamp_units"`
```

Comment:
Do you really support other timestamp units? If so use
```suggestion
	TimestampUnits config.Duration   `toml:"timestamp_units"`
```